tests/quasi-newton/helpers.cpp

```diff
@@ -202,4 +202,79 @@ void runTestQNEmptyPartition(std::string const &config, TestContext const &conte
   }
 }
 
+void runTestQNBoundedValue(std::string const &config, TestContext const &context)
+{
+  std::string meshName, writeDataName, readDataName;
+
+  if (context.isNamed("SolverOne")) {
+    meshName      = "MeshOne";
+    writeDataName = "Data1";
+    readDataName  = "Data2";
+  } else {
+    BOOST_REQUIRE(context.isNamed("SolverTwo"));
+    meshName      = "MeshTwo";
+    writeDataName = "Data2";
+    readDataName  = "Data1";
+  }
+
+  precice::Participant interface(context.name, config, context.rank, context.size);
```

Comment:
```suggestion
  precice::Participant participant(context.name, config, context.rank, context.size);
```
With new tests, we could directly use the new name. "precice" is OK as well, but "interface" comes from the old "SolverInterface".